### PR TITLE
Fixing undo text-annotations in iOS

### PR DESCRIPTION
**Pull request by marinaserranomontes**
## Pull request checklist

- [ ] All tests pass. Demo project builds and runs.
- [ ] I have resolved any merge conflicts. Confirmation: ____

## What's in this pull request?

>...

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/opentok/annotation-acc-pack/pull/121" target="_blank">ReviewNinja

File: js/opentok.js-annotations/src/annotation-widget.js [hunk 1]
```diff
@@ -1058,7 +1058,7 @@
         historyItem = drawHistory[i];
         if (historyItem.fromId === cid) {
 
-          if(historyItem.platform === 'ios' && itemsToRemove != null && itemsToRemove.length > 0) {
+          if(historyItem.platform === 'ios' && itemsToRemove != null && itemsToRemove.length > 0 && itemsToRemove[0] != null) {
```
Comment (adrice727) [inline js/opentok.js-annotations/src/annotation-widget.js]: Change we change this line to:

`(historyItem.platform === 'ios' && !!itemsToRemove && !!itemsToRemove.length && itemsToRemove[0] !== null)`

Comment (marinaserranomontes) [inline js/opentok.js-annotations/src/annotation-widget.js]: +1 @adrice727  thanks!!

**Comment by adrice727** [comment]
👍

**Pull request closed without merging**